.changeset/long-hornets-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/firestore": patch
```

Comment:
Do we also need a patch for the firebase-wide package?

Reply:
I believe that we don't need to for a patch release, since every release is at least a minimum patch version bump.